htdocs/reception/class/reception.class.php

```diff
@@ -137,8 +139,55 @@ class Reception extends CommonObject
 	// We can use this to know warehouse planned to be used for each lot.
 	public $detail_batch;
 
+	/**
+	 * Workflow of the status variables:
+	 *
+	 * A reception is first created as a draft. It can then be validated
+	 * to apply the changes in the stock, and the closed when the stock
```

Comment:
With develop, it is the workflow module that decide when the stock is modified.
Can be when validating reception and can be when closing reception.

Reply:
Thanks for the quick review and the pointer, I'll check the path more in details and address that in the week end.

Reply:
> With develop, it is the workflow module that decide when the stock is modified.

After some research, I'm not sure I understand this, the Reception class is not calling anything related to the Workflow module, and vice-versa, and it creates the MouvementStock entries. Is it related to future work?